daemon/Makefile

```diff
@@ -40,3 +41,9 @@ install-binary: install
 install-bash-completion:
 
 endif
+
+../pkg/datapath/loader/datapath_sha256.go: ../tools/generate-cilium-datapath-sha256/main.go $(BPF_SRCFILES_REL)
+	if [ "${GOARCH}" = "amd64"]; then \
```

Comment:
Any reason this would not work on `arm64` as well?

Reply:
Related, how does this not break the arm64 build?

Reply:
:fearful: oh, because it's committed to the tree.

If we fail to update this datapath sha file, then it can break upgrades (basically if the datapath sha seems to be the same then there's no datapath changes therefore no need to replace the BPF programs). Let's be super careful here.